SimPEG/Examples/DC_Forward_PseudoSection.py

```diff
@@ -74,7 +74,13 @@ def run(loc=None, sig=None, radi=None, param=None, surveyType='dipole-dipole', u
 
     # We will handle the geometry of the survey for you and create all the combination of tx-rx along line
     # [Tx, Rx] = DC.gen_DCIPsurvey(locs, mesh, surveyType, param[0], param[1], param[2])
-    survey, Tx, Rx = DC.gen_DCIPsurvey(locs, mesh, surveyType, param[0], param[1], param[2])
+    srcList = StaticUtils.gen_DCIPsurvey(locs, mesh, surveyType, param[0], param[1], param[2])
+    print(srcList)
+
+    raise Exception('The output of the function changed. We need to create the survey and get Rx to continue.')
```

Comment:
@lheagy : Yea it s the best way right now. I will keep it local and bring it back in once I find the time to fix it.
@seogi : I think you and I already talked about this. You can remove from PR